domain/schema/controller/sql/0024-agent-binary-metadata.sql

```diff
@@ -0,0 +1,19 @@
+-- The agent_binary_metadata table in the controller database records information about
+-- cached agent binaries, including their version, SHA, and object store location.
```

Comment:
'cached' is a loaded term.
Just need to say that the agent binaries are stored in the controller's object store.

There's an object store uuid column but the comment refers to "location" which to me implies a path of some sort.